### DISCUSSION TITLE
[BUG] ¿Es posible hacer copy de un texto de algun canal desde la version mobile?

### DISCUSSION THREAD

**Discussion by jdurandpaz** [Cotalker/documentation#122] [4. Bug Reports] [answered]
Estimados,

Estuve intentando hacer copy de un mensaje ( en cotalker android e iOS versión 15.5.5) de un canal para probar un form externo y no pude realizarlo en Android ni iOS. No se si es algo que no esta desarrollado o un bug.

Saludos

**Comment by acedward** [comment, marked as answer]
@jdurandpaz suena como un bug, los mensaje deberian ser poder ser copiados. Vamos a revisarlo.

**Reply by jdurandpaz** [reply]
vale avísame y lo creo como bug si aplica

**Reply by jdurandpaz** [reply]
@acedward consulta necesitas que cree el ticket o ya lo estan revisando

**Reply by acedward** [reply]
Ya está en el backlog. Siempre lo que encontremos aquí se reportará automáticamente.

**Reply by acedward** [reply]
UPDATE: Se liberará pronto, está en desarrollo
